src/views/blocks/tickets/submit-button.php

```diff
@@ -23,14 +26,27 @@
  * @param string $button_name The button name. Set to cart-button to send to cart on submit, or set to checkout-button to send to checkout on submit.
  */
 $button_name = apply_filters( 'tribe_tickets_ticket_block_submit', 'cart-button' );
+
+/**
+ * Allow filtering of the button classes for the tickets block.
+ *
+ * @since TBD
+ *
+ * @param array $button_name The button classes.
+ */
+$button_classes = apply_filters( 'tribe_tickets_ticket_block_submit_classes', [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Short array syntax is not allowed

<sub>hash: 72fb5f183b852113b377edde60fcdf8c</sub>